TappNetwork/home-visiting-app-web
# Cu 3r59dgn notifications

# Details
This PR introduces notifications to the HV App. Notifications will be issued whenever a user has unread messages in a thread, new announcements are made, or a new internal resource is published. Notifications are accessed in the front end via a bell icon that appears when notifications are present. 

# Steps to test changes

1. follow steps in backend PR
2. Login to front end.
3. ensure count of notifications inside icon is centered
4. click notifications icon
5. click download link for resource notification 
6. click messenger, modal should close and messenger should open
7. hard refresh page
8. Grayed out notification icon should display
9. In DB, null seen_at column for a few notifications
10. Refresh page
11. ensure number centered for single digit counts

# Issues Addressed

[3r59dgn](https://app.clickup.com/t/3r59dgn)

# Screenshots

<img width="489" alt="Screen Shot 2022-11-02 at 12 10 25 PM" src="https://user-images.githubusercontent.com/29612767/199597984-ad76701b-f1dd-4844-802c-ed5384834f87.png">
<img width="488" alt="Screen Shot 2022-11-02 at 12 10 08 PM" src="https://user-images.githubusercontent.com/29612767/199598002-a89d567e-1dc5-4604-a826-08b951084fc7.png">
